ci-operator/jobs/openshift/cluster-api-provider-libvirt/openshift-cluster-api-provider-libvirt-master-presubmits.yaml

```diff
@@ -69,7 +69,7 @@ presubmits:
         - name: CONFIG_SPEC
           valueFrom:
             configMapKeyRef:
-              key: openshift-installer-master.yaml
+              key: openshift-cluster-api-provider-libvirt-master.yaml
```

Comment:
/hold

@petr-muller @droslean @bbguimaraes how did this happen in the first place?

Reply:
https://github.com/openshift/release/pull/4047

It shouldn't....

Reply:
The job was created manually obviously and we didn't validate that `cm`.

Reply:
Heh, first refresh, then comment.